saga/fx/put.ts

```diff
@@ -0,0 +1,12 @@
+import { action } from "../../instructions.ts";
+
+import { ActionContext } from "../context.ts";
+import type { Action } from "../types.ts";
+
+export function put(act: Action) {
+  return action(function* (resolve) {
+    const { input } = yield* ActionContext;
+    input.send(act);
```

Comment:
`send()` is an `Operation`, so I think you want to evaluate ite:
```suggestion
    yield* input.send(act);
```